src/modules/src/Kconfig

```diff
@@ -165,6 +165,15 @@ config MOTORS_REQUIRE_ARMING
         When enabled, system must be armed to be able to take off. Arming can
         be done in several ways, e.g. though cfclient or external transmitter.
 
+config MOTORS_DEFAULT_IDLE_THRUST
+    int "Idle thrust for the armed motors"
+    depends on MOTORS_REQUIRE_ARMING
+    range 0 65535
```

Comment:
Maybe we should limit the range to say 15000 as more then this should "never" be needed?